docs/api-reference/geo-layers/tile-layer.md

```diff
@@ -114,6 +114,7 @@ If supplied, `getTileData` is called to retrieve the data of each tile. It recei
 - `z` (Number) - z index of the tile
 - `url` (String) - resolved url of the tile if the `data` prop is provided, otherwise `null`
 - `bbox` (Object) - bounding box of the tile. When used with a geospatial view, `bbox` is in the shape of `{west: <longitude>, north: <latitude>, east: <longitude>, south: <latitude>}`. When used with a non-geospatial view, `bbox` is in the shape of `{left, top, right, bottom}`.
+- `signal` (Object) - an [AbortSignal](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal) that may be signalled if there are too many queued requests. Note: only tiles that aren't visible will be aborted.
```

Comment:
It might be good to add in the docs somewhere that a custom `getTileData` can check `signal.aborted` manually. So it can do:

```js
const {signal} = tile;
const data = fetch(url, {signal});

if (signal.aborted) {
   return null;
}
// Expensive computation on returned data
```

I believe with this PR, if a tile is aborted, the response is never cached? This might hurt performance in a situation where the user doesn't implement `signal`, i.e. doesn't pass it to `fetch`. In this case, the entire `fetch` and data processing would still happen but since the `TileLayer` instigated an `abort`, the `TileLayer` wouldn't cache the response. Maybe there's a way to abort the `getTileData`, but _if_ it still returns a non-null response, then cache it?

Reply:
Awesome suggestions. Nice.